testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/epcpropagation/unsync/BMTEPCStatefulBean.java

```diff
@@ -96,7 +96,7 @@ public void allowjoinedunsyncPersistenceXML()  throws SystemException, NotSuppor
         UserTransaction userTxn = sessionContext.getUserTransaction();
         userTxn.begin();
         try {
-            //Join Transaction to force unsynchronized persistence context to be associated with jta tx
+            //Join Transaction to force unsynchronized persistence context to be associated with Jakarta Transactions tx
```

Comment:
`Jakarta Transactions tx` -> `the Jakarta Transaction`

Reply:
going with `Jakarta Transactions tx`